WcaOnRails/spec/factories/users.rb

```diff
@@ -29,36 +29,43 @@
       name "Mr. Admin"
       email "admin@worldcubeassociation.org"
       after(:create) do |user|
-        software_team = Team.find_by_friendly_id('wst')
+        software_team = Team.wst
         FactoryBot.create(:team_member, team_id: software_team.id, user_id: user.id, team_leader: true)
       end
     end
 
+    factory :board_member, traits: [:wca_id] do
+      after(:create) do |user|
+        board_team = Team.board
+        FactoryBot.create(:team_member, team_id: board_team.id, user_id: user.id)
+      end
+    end
+
     trait :wrt_member do
       after(:create) do |user|
-        results_team = Team.find_by_friendly_id('wrt')
+        results_team = Team.wrt
```

Comment:
I think it would be fine to combine these two to one line:

```ruby
FactoryBot.create(:team_member, team_id: Team.results.id, user_id: user.id)
```

(same for every other occurrence in this file)